emailUtil.ts

```diff
@@ -0,0 +1,53 @@
+import nodemailer from "nodemailer";
+import config from "./config";
+import { Recording } from "./models/Recording";
+import { TrackTag } from "./models/TrackTag";
+import log from "./logging";
+import moment, { Moment } from "moment";
+
+function alertHTML(
+  recording: Recording,
+  tag: TrackTag,
+  camera: String
+): string {
+  const dateTime = moment(recording.recordingDateTime).format(" H:MMa Do MMM");
+  let html = `<h1>${camera} has detected a ${tag.what} - ${dateTime}</h1>`;
+  html += `<a  href="${config.server.recording_url_base}/${recording.id}?device=${recording.DeviceId}">View Recording</a>`;
+  html += "<br><p>Thanks,<br> Cacophony Team</p>";
+  return html;
+}
+
+async function sendEmail(
+  html: string,
```

Comment:
Do we need a text only representation of the email too, or does nodemailer generate one automatically from the html version?